deployment/v3/utils/prop_migrator/knowledge/property-categories.csv

```diff
@@ -1,4 +1,760 @@
 Property file name,Key,Property Category
+admin-default.properties,authmanager.base.url,URL
+admin-default.properties,mosip.admin.accountmgmt.auth-manager-base-uri,URL
+admin-default.properties,mosip.kernel.signature.cryptomanager-encrypt-url,URL
+admin-default.properties,auth.server.validate.url,URL
+admin-default.properties,auth.server.refreshToken.url,URL
+admin-default.properties,javax.persistence.jdbc.url,URL
+admin-default.properties,javax.persistence.jdbc.password,Key
+admin-default.properties,auth.server.user-register-url,URL
+admin-default.properties,mosip.kernel.emailnotifier-url,URL
+admin-default.properties,auth.server.sendotp-url,URL
+admin-default.properties,auth.server.user-add-password-url,URL
+admin-default.properties,mosip.admin.security.policy.userrole-auth-url,URL
+admin-default.properties,mosip.admin.uinmgmt.uin-detail-search,URL
+admin-default.properties,mosip.mosip.kernel.packet-status-update-url,URL
+admin-default.properties,mosip.kernel.packet-reciever-api-url,URL
+admin-default.properties,mosip.kernel.zone-validation-url,URL
+admin-default.properties,mosip.kernel.audit.manager.api,URL
+admin-default.properties,mosip.kernel.masterdata.audit-url,URL
+admin-default.properties,mosip.admin-services.audit.manager.api,URL
+admin-default.properties,mosip.open-id.base-url,URL
+admin-default.properties,mosip.admin-services.open-id,clientsecret,Key
```

Comment:
Check this it has comma before and after the clientsecret

Reply:
updated sir